src/Exporter.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter;
+
+use Exporter\Source\SourceIteratorInterface;
+use Exporter\Writer\CsvWriter;
+use Exporter\Writer\JsonWriter;
+use Exporter\Writer\TypedWriterInterface;
+use Exporter\Writer\XlsWriter;
+use Exporter\Writer\XmlWriter;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+
+class Exporter
+{
+    /**
+     * @var TypedWriterInterface[]
+     */
+    private $writers;
+
+    /**
+     * NEXT_MAJOR: change default value to array().
```

Comment:
I copied this from a PR I made on the core bundle, but I think you're right : this is new, so no need for deprecation at all… people who change from one class to another will know what to do.